public_html/wp-content/plugins/multi-event-sponsors/classes/mes-sponsor.php

```diff
@@ -391,7 +391,7 @@ protected function save_post_meta( $post_id, $new_values ) {
 
 		$text_fields = array(
 			'company_name', 'website', 'first_name', 'last_name', 'email_address', 'phone_number', 'twitter_handle',
-			'street_address1', 'street_address2', 'city', 'state', 'zip_code', 'country',
+			'street_address1', 'street_address2', 'city', 'state', 'zip_code', 'country', 'first_time',
```

Comment:
Thanks for the context 👍 I've removed it in e7e6481e20e8a08d015200a73f9073a863cb8c53.
And set the value `no` in `get_stub_me_sponsors_meta()` in 05f2f750661a6ca1fa8c7b6ad4ba2ae41b8401c8.

But for ^, I'm a bit not sure how we usually test it. To my understanding, the sponsors in the MES post type on Central seem always to be added to individual sites, and I guess they would be added automatically when an individual site is set up. So is building a new site a common way to test the commit here? (If it's not the common way, what would it be? And at the same time, I'm curious, is the process of automatically creating an individual WordCamp site something I can test in a sandbox?

Reply:
Yeah, you'd need to create a new site to test it. It's not good to do that on your w.org sandbox, since that changes the production database. I'd do it in a local Docker env instead.

In mine I have a WordCamp post called "new site", and I've moved it through the various post statuses so that it has the "needs site" status. Then you can fill in the URL field and check the `create site` box, and update the post. That'll automatically copy the MES posts to the site.

It you wanna repeat it, you can `wp site delete {site ID}` and then check the box again.

Reply:
> In mine I have a WordCamp post called "new site", and I've moved it through the various post statuses so that it has the "needs site" status. Then you can fill in the URL field and check the create site box, and update the post. That'll automatically copy the MES posts to the site.
It you wanna repeat it, you can wp site delete {site ID} and then check the box again.

Much appreciated for sharing this 🙏 I've successfully done testing locally and can see that all copied MES posts have their first_time value set to `no`.